lib/fluri.dart

```diff
@@ -271,11 +271,11 @@ class FluriMixin {
       // Add the param value(s) while eliminating duplicates.
       // Throw an ArgumentError if any value is invalid.
       if (value is String && !newQueryParameters.containsValue(value)) {
```

Comment:
Totally unrelated, but I think this is a bug and should be `!newQueryParameters[key].containsValue(value)`..

Reply:
Pushed a commit with a test to verify this is a bug and another commit to fix it.